src/core/opamStubs.mli

```diff
@@ -0,0 +1,130 @@
+(**************************************************************************)
```

Comment:
For this (and other added modules) could you also add a line in `/doc/index.html` ? Thanks